### PR TITLE
fix: update s51 logic to check if first or last name exists

### PR DESCRIPTION
**Pull request by cprieststephens**
## Describe your changes
[APPLICS 1017](https://pins-ds.atlassian.net/browse/APPLICS-1017): Correctly display firstName or lastName on s51 advice page 
- Updated conditional logic so that first or last name can be rendered individually if they exist

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/section-51/index/_utils/advice-helpers.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ const getAdviceName = ({ organisation, firstName, lastName }, i18n) => {
 	let name = i18n.t('section51.anonymous');
 	if (organisation?.trim()) name = organisation.trim();
```
Comment (Philip-Philippou1) [inline packages/forms-web-app/src/pages/projects/section-51/index/_utils/advice-helpers.js:12]: Not critical but would something like this be consistent with others?

const getAdviceName = ({ organisation, firstName, lastName }, i18n) => {
	return  organisation?.trim() ||
		(firstName && lastName ? `${firstName} ${lastName}`
		: firstName || lastName || i18n.t('section51.anonymous'));
};

**Review by emil-placheta — APPROVED** [review]

**Pull request merged**